### PR TITLE
docs - update publishing details, add menuTitle instructions

### PR DESCRIPTION
**Pull request by robbymilo**
**What this PR does / why we need it**:

- Adds instructions for using the `menuTitle` front matter param
- Updates deploy information

### CODE REVIEW

**Review by achatterjee-grafana — APPROVED** [review]
lgtm

File: docs/README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ If you are copying an existing page as the basis for a new one, be sure to remov
 
 The side menu is automatically build from the file structure. Use the [weight](https://gohugo.io/templates/lists/#by-weight) front matter parameter to order pages.
 
+Use the front matter param `menuTitle` to specify different menu text from the page title.
```
Comment (osg-grafana) [inline docs/README.md:48]:
```suggestion
To specify different menu text from the page title, use the front matter parameter `menuTitle`.
```

Comment (osg-grafana) [reply docs/README.md:48]: Nit: Put the _what_ before _how_.

File: docs/README.md [hunk 2]
```diff
@@ -53,6 +55,9 @@ Images are currently hosted in the grafana/website repo.
 
 ## Deploy changes to grafana.com
 
-When a PR is merged to main with changes in the `docs/sources` directory, those changes are automatically synced to the grafana/website repo and published to the staging site.
+When a PR is merged with changes in the `docs/sources` directory, those changes are automatically synced by a GitHub action (`.github/workflows/publish.yml`) to the grafana/website repo.
+
+* A PR targeting the `main` branch will sync to the `content/docs/grafana/next` directory in the website repo, and publish to https://grafana.com/docs/grafana/next/.
```
Comment (osg-grafana) [inline docs/README.md:60]:
```suggestion
* A PR that targets the `main` branch syncs to the `content/docs/grafana/next` directory in the `website` repository, and publishes to `https://grafana.com/docs/grafana/next/`.
```

File: docs/README.md [hunk 3]
```diff
@@ -53,6 +55,9 @@ Images are currently hosted in the grafana/website repo.
...
+* A PR targeting the latest/current release branch will sync to the `content/docs/grafana/latest` directory in the website repo, and publish to https://grafana.com/docs/grafana/latest/.
```
Comment (osg-grafana) [inline docs/README.md:61]:
```suggestion
* A PR targeting the `latest/current` release branch syncs to the `content/docs/grafana/latest` directory in the `website` repository, and publishes to `https://grafana.com/docs/grafana/latest/`.
```

File: docs/README.md [hunk 4]
```diff
@@ -53,6 +55,9 @@ Images are currently hosted in the grafana/website repo.
...
 
-Generally, someone from marketing will publish to production each day: so as long as the sync is successful your docs edits will be published. Alternatively, you can refer to [publishing to production](https://github.com/grafana/website#publishing-to-production-grafanacom) if you'd like to do it yourself.
+Once the sync is complete, the website will automatically publish to production - no further action is needed.
```
Comment (osg-grafana) [inline docs/README.md:63]:
```suggestion
After the sync is complete, the website will automatically publish to production.
```
What does "the website will automatically publish to production" mean? Where is that happens?

**Review by osg-grafana — CHANGES_REQUESTED** [review]

**Review by osg-grafana — APPROVED** [review]

**Pull request merged**